### ISSUE TITLE
minimos-quadrados-marcela-roberto-et_all

### ISSUE THREAD

**Issue by Marcelarclaudio** [mat-esp-2015/minimos-quadrados-marcala-roberto-et_all#1]
https://github.com/mat-esp-uerj/minimos-quadrados-marcala-roberto-et_all.git

**Comment by robertosouza5991** [comment]
Marcela, a nossa turma não precisava fazer esta tarefa pois não tivemos 
a aula.

Mas claro que podemos fazer e entregar até amanha, mas a prioridade é 
para a ultima tarefa.

Em 13/01/2016 23:13, Marcelarclaudio escreveu:

**Comment by victortxa** [comment]
@Marcelarclaudio @robertosouza5991  o issue deve ser aberto aqui: https://github.com/leouieda/matematica-especial/issues e não no repositório da prática.

**Comment by Marcelarclaudio** [comment]
Sim, mas no caso essa de minimos quadrados me confundi, não foi feita pela minha turma!
Date: Thu, 14 Jan 2016 04:54:36 -0800
